src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/Modal/view_all_messages_modal.html.twig

```diff
@@ -30,7 +30,12 @@
         <div class="row">
           <div class="col">
             <h3 class="card-header-title">
-              {{ 'Message history with %name%'|trans({'%name%': orderForViewing.customer.firstName ~ ' ' ~ orderForViewing.customer.lastName}, 'Admin.Global') }} ({{ orderForViewing.messages.messages|length }})
+              {% if orderForViewing.customer is not null %}
+                {{ 'Message history with %name%'|trans({'%name%': orderForViewing.customer.firstName ~ ' ' ~ orderForViewing.customer.lastName}, 'Admin.Global') }}
```

Comment:
Is this string necessary? What will %name% indicate? We are talking about a deleted customer, we are not supposed to have its name anymore.

Reply:
Actually, "Message history with %name%" is the sentence when the customer is not deleted. %name% will "firstname lastname". When the customer is deleted, the sentence will be "Message history" as you can see just below. 🙂